packages/api-server/src/cliConfig.ts

```diff
@@ -0,0 +1,72 @@
+import type { Argv } from 'yargs'
+
+import type { APIParsedOptions, BothParsedOptions } from './types'
+
+const apiDescription = 'Start a server for serving only the api side'
+
+function apiBuilder(yargs: Argv<APIParsedOptions>) {
+  yargs.options({
+    port: {
+      description: 'The port to listen at',
+      type: 'number',
+      alias: 'p',
+    },
+    host: {
+      description:
+        "The host to listen at. Note that you most likely want this to be '0.0.0.0' in production",
+      type: 'string',
+    },
```

Comment:
I deliberately didn't add any defaults here. I'm starting to feel like we shouldn't specify defaults using yargs. It's information if the user didn't specify the flag and we do things differently if they don't. With a default, we can't tell if they specified the flag or if it's just the default